src/main/java/edu/kis/powp/jobs2d/command/TxtParser.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * txt file in format:
+ * Z XX YY
+ * Z XX YY
+ * where:
+ * Z - 's' for setPosition or 'o' for operateTo
+ * XX - x coordinate (int)
+ * YY - y coordinate (int)
+ **/
+public class TxtParser implements CommandParser {
+
+	private String fileContent;
+	private List<DriverCommand> commandList;
+
+	public TxtParser(String fileContent) {
+		this.fileContent = fileContent;
+		this.commandList = new ArrayList<>();
+	}
+
+	public void fillListFromFile() {
```

Comment:
is it really important that string is from a file? Couldn't it be the contents of some URL or a textbox?

Reply:
fixed